common/containers/Tabs/Swap/components/SwapInfoHeader.js

```diff
@@ -0,0 +1,139 @@
+import React, { Component } from 'react';
+import { toFixedIfLarger } from 'utils/formatters';
+import translate from 'translations';
+import * as swapTypes from 'actions/swapTypes';
+
+export type ReduxStateProps = {
+  timeRemaining: string,
+  originAmount: string | string,
+  originKind: string,
+  destinationKind: string,
+  destinationAmount: string | number,
+  referenceNumber: string
+};
+
+export type ReduxActionProps = {
+  restartSwap: () => swapTypes.RestartSwapAction
+};
+
+export default class SwapInfoHeader extends Component {
+  props: ReduxStateProps & ReduxActionProps;
+
+  computedOriginDestinationRatio = () => {
+    return toFixedIfLarger(
+      this.props.destinationAmount / this.props.originAmount,
+      6
+    );
+  };
+
+  isExpanded = () => {
+    const { referenceNumber, timeRemaining, restartSwap } = this.props;
+    return referenceNumber && timeRemaining && restartSwap;
+  };
+
+  computedClass = () => {
+    if (this.isExpanded()) {
+      return 'col-sm-3 order-info';
+    } else {
+      return 'col-sm-4 order-info';
+    }
+  };
+
+  render() {
+    const {
+      referenceNumber,
+      timeRemaining,
+      originAmount,
+      destinationAmount,
+      originKind,
+      destinationKind,
+      restartSwap
+    } = this.props;
+    return (
+      <div>
+        <section className="row text-center">
+          <div className="col-xs-3 text-left">
+            <button className="btn btn-danger btn-xs" onClick={restartSwap}>
+              Start New Swap
+            </button>
+          </div>
+          <h5 className="col-xs-6">
+            {translate('SWAP_information')}
+          </h5>
+          <div className="col-xs-3">
+            <a
+              className="link"
+              href="https://bity.com/af/jshkb37v"
```

Comment:
Maybe this should be a constant somewhere, since it also shows up in the footer.